### PR TITLE
Change password strength message

### LINKED ISSUE #1039: Copy: Password field messaging
Design new copy for the password field strength indicator, research examples as needed.

### PR DESCRIPTION
**Pull request by na3d**
Closes #1039

### CODE REVIEW

File: src/components/form/PasswordInput.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component, PropTypes } from 'react';
 
 import Input from './Input';
+const str = ['Extremely Weak', 'Very Weak', 'Weak', 'Moderate', 'Strong'];
```
Comment (eatonphil) [inline src/components/form/PasswordInput.js]: This seems pretty heavy on the weak side (e.g. where is 'Extremely strong'?). But maybe this is the more appropriate list anyway?

Comment (na3d) [inline src/components/form/PasswordInput.js]: I was trying not to encourage a password I wouldn't use myself on a public facing system.  When typing in the different passwords I use, this seems most appropriate.

**Comment by eatonphil** [comment]
Screenshot of each stage?

File: src/components/form/PasswordInput.js [hunk 2]
```diff
@@ -43,9 +43,7 @@ export default class PasswordInput extends Component {
           <span></span>
         </div>
         {this.state.password !== '' ? <div className="PasswordInput-cracktime">
-          An offline attack would
-          take {this.state.strength.crack_times_display[passwordType]} to
-          crack this password.<br />
+          {str[this.state.strength.score]}
```
Comment (abemassry) [inline src/components/form/PasswordInput.js]: this.state.strength.score returns 0-4? or could we define more granularity

Comment (na3d) [inline src/components/form/PasswordInput.js]: Yeah, it returns 0-4.

File: src/components/form/PasswordInput.js (hunk 1 above)
Comment (mparke) [inline src/components/form/PasswordInput.js]: +1 to heavy on the weak side, only suggestion is there a better word than `Extremely`?

also, better variable name here? I should know what I'm referencing below.

**Comment by na3d** [comment]
screen shot 2017-01-24 at 11 36 46 am
screen shot 2017-01-24 at 11 36 57 am
screen shot 2017-01-24 at 11 37 09 am
screen shot 2017-01-24 at 11 37 18 am
screen shot 2017-01-24 at 11 37 27 am

File: src/components/form/PasswordInput.js (hunk 1 above)
Comment (eatonphil) [inline src/components/form/PasswordInput.js]: Whoops, saw this after I merged it. If this is still an issue, could you open an issue or just make a one-word PR with a suggestion?

**Pull request merged**